archtec/scripts/faq_content.lua

```diff
@@ -13,240 +13,294 @@ archtec.faq.register("Namecolor", {
 
-Join/create a channel: (Add 'yes' to your command to make new the created channel public)
-'/c join <channel>' or
-'/c j <channel>'
+Required arguments/parameters are indicated with angle braces <> while optional ones are have curly braces {}
+
+Join/Create a channel: (Add 'yes' to your command to make new the created channel public)
+'/c join <channel> {yes}' or
+'/c j <channel> {yes}'
 
 Leave a channel:
 '/c leave <channel>' or
 '/c l <channel>'
 
-Invite someone in a channel:
+Invite someone to a channel:
 '/c invite <channel> <name>' or
 '/c i <channel> <name>'
 
 List all channels:
 '/c list' or
 '/c li'
 
-Find all channels where someone is:
+List all channels a player is currently in:
 '/c find <name>' or
 '/c f <name>'
 
-Kick someone from a channel. Can be used by channelowners and staff:
+Kick a player from a channel. Can only be used by channel owners and staff:
 '/c kick <channel> <name>' or
 '/c k <channel> <name>'
 
-Get help:
-'/c help <command>' or
-'/c h <command>' (command is optional)
+Show help:
+'/c help {command}' or
+'/c h {command}'
 ]]
 })
 
 archtec.faq.register("Chatbridge to Discord", {
     header = "Chat",
     pos = 4,
     description = [[
-Since we use Matterbridge, we might add support for new platforms later.
-
-The bridge supports the following commands and chat formats:
-- Normal chat messages
-- '/me' Shows ingame what you're doing (formatted like regular '/me')
-- '!status' Shows the server status (like '/status' ingame)
+The Discord chart bridge uses Matterbridge facilitating additional platform
+    integrations in the future.
+
+The following commands and chat formats are supported:
+- Regular chat messages
+- '/me' Used in conjunction with a verb do indicate you as a player performing
+    an action as described by a 3rd person, e.g. /me talks -> Player talks
+- '!status' Shows the server status
 - '!cmd' Remote command execution (staff only)
 ]]
 })
 
--- Player interaction
-archtec.faq.register("Ignoring", {
+-- Player blocking
+archtec.faq.register("Ignore", {
     header = "Player interaction",
     pos = 1,
     description = [[
-You can ignore other players resp. other players can ignore you. This means you can no longer act together.
-Not all functions on the server support ignoring, but most do :).
+Allows you to block other players from interacting and communicating with you.
+Currently ignore isn't supported by all features/functions but most are. :).
 
-Ignore a player:
+Block/Ignore a player:
 '/ignore add <name>' or
 '/ignore ignore <name>'
 
-Unignore a player:
+Unblock/Un-ignore player:
 '/ignore remove <name>' or
 '/ignore unignore <name>'
 
-List ignored players:
+List blocked/ignored players:
 '/ignore list' or
-'/ignore list <name>' (lists ignored players of <name>; staff only)
-'/ignore' (without params)
+'/ignore list {name}' (lists ignored players of <name>; staff only)
```

Comment:
{} should be explained in the FAQ default description using `archtec.faq.default_description()`

Reply:
(my job)